src/server.ts

```diff
@@ -143,7 +146,15 @@ export async function startServer<RawConfiguration, Configuration, State>(
       },
     },
     (_request: FastifyRequest): CapabilitiesResponse => {
-      return connector.getCapabilities(configuration);
+      const capabilitiesResponse = tracer.startActiveSpan(
+        "getCapabilities",
+        (span) => {
+          const capabilitiesResponse = connector.getCapabilities(configuration);
+          span.end();
+          return capabilitiesResponse;
+        }
+      );
```

Comment:
I think I remember seeing some fastify middleware that wrapped the endpoints for OTEL automatically. Would this be viable? No worries if you want to keep it manual.